reverse_strand.py

```diff
@@ -0,0 +1,39 @@
+#! /home/juliette/miniconda3/bin/python3
```

Comment:
This shebang is not going to be supported on any other machine than your own one; I suggest using the most universal one, `#! /usr/bin/env python3`